src/test/ui/moves/move-fn-self-receiver.stderr

```diff
@@ -0,0 +1,34 @@
+error[E0382]: use of moved value: `val.0`
+  --> $DIR/move-fn-self-receiver.rs:11:5
+   |
+LL |     val.0.into_iter().next();
+   |     ----------------- value moved due to this method call
+LL |     val.0;
+   |     ^^^^^ value used here after move
```

Comment:
*If* we can get the span for the method call itself, then this can be

```
error[E0382]: use of moved value: `val.0`
  --> $DIR/move-fn-self-receiver.rs:11:5
   |
LL |     val.0.into_iter().next();
   |     ----- ----------- value moved due to this method call
   |     |
   |     value moved here
LL |     val.0;
   |     ^^^^^ value used here after move
```

Reply:
Although looking at the mockup it seems like it might be too much for no gain. Feel free to disregard. I was originally worried about long chains (`foo.bar().baz().bat()`) having long spans, but the visual output will _always_ be covering everything, whether it is one span or two.